docs/user/settings.rst

```diff
@@ -411,6 +411,49 @@ authorize the request with other freeradius modules.
 Set this to ``True`` if you are performing authorization exclusively
 through the REST API.
 
+.. _openwisp_radius_additional_radius_checks:
+
+``OPENWISP_RADIUS_ADDITIONAL_RADIUS_CHECKS``
```

Comment:
I don't understand what's the point of this setting. What's the point? Why does the user need to set this?